apps/web/components/connected/AdministrationOfOccupationalSafetyAndHealthCourses/AdministrationOfOccupationalSafetyAndHealthCourses.tsx

```diff
@@ -22,12 +22,18 @@ const normalizesAndMatch = (value1: string, value2: string) => {
 
 interface AdministrationOfOccupationalSafetyAndHealthCoursesProps {
   slice: ConnectedComponent
+  title: string | null
+  headingLevel: 'h2' | 'h3' | 'h4'
+  hasBorderAbove: boolean
 }
 
 type ListState = 'loading' | 'loaded' | 'error'
 
 const AdministrationOfOccupationalSafetyAndHealthCourses = ({
   slice,
+  title = slice.json?.title ?? null,
+  headingLevel = slice.configJson?.headingLevel ?? 'h2',
```

Comment:
Do we need these props to be here, since we won't be using them could we skip having them?